services/port_manager/src/main/resources/application.properties

```diff
@@ -17,12 +17,4 @@ microservices.ip.service.url=http://localhost:9004/ips
 microservices.mac.service.url=http://localhost:9005/macs
 microservices.sg.service.url=http://127.0.0.1:8080/securitygroups
 microservices.route.service.url=http://localhost:9003/routes
-microservices.node.service.url=http://localhost:9007/nodes
-#MyBatis configuration
-spring.datasource.driverClassName=org.apache.ignite.IgniteJdbcThinDriver
-spring.datasource.url=jdbc:ignite:thin://127.0.0.1
-spring.datasource.schema=classpath:sql/port.sql
-mybatis.configuration.map-underscore-to-camel-case=true
-#spring.datasource.username=your_username
-#spring.datasource.password=your_password
-logging.level.com.dao=debug
\ No newline at end of file
+microservices.node.service.url=http://localhost:9007/nodes
```

Comment:
Need to add dpm service url as in commit https://github.com/futurewei-cloud/alcor/pull/244/commits/5fce67884499629dc754f192b6d02f86b77b5c4d

Let me add it for you to kick off the integration.